CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.11.0
+------------------------------
+*November 6, 2018*
+
+### Added
+- Added `display: none` to `has-noItems` within the listing component.
+
```

Comment:
Super (Mega) NIT: Should be double spacing between entries (needs some for a couple of entries below this too)